Automattic/woocommerce-payments
# fix: external link text

#### Changes proposed in this Pull Request









The "Opens in a new tab" text is meant to be visible to screen readers only. But instead, it is visible to everyone. Fixing.

Before:
![Screen Shot 2023-02-16 at 10 45 50 AM](https://user-images.githubusercontent.com/273592/219459999-08c7a014-9588-4118-92ba-ea9f9780bad3.png)

After:
![Screen Shot 2023-02-16 at 10 45 45 AM](https://user-images.githubusercontent.com/273592/219459970-a6d1a2f8-1167-4ffd-9d39-513843e1e62d.png)

#### Testing instructions


- Go to the settings page: `/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments`
- "(opens in a new tab)" text is no longer visually displayed



*

-------------------


**Post merge**
